Playbooks/playbook-Detonate_File_-_Generic.yml

```diff
@@ -0,0 +1,158 @@
+id: Detonate File - Generic
+version: -1
+name: Detonate File - Generic
+description: Detonate file through active integration that support file detonation
+fromversion: "3.5.0"
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 73eadb3f-61fa-41d5-87b3-6d6a67a04e63
+    type: start
+    task:
+      id: 73eadb3f-61fa-41d5-87b3-6d6a67a04e63
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      '#none#':
+      - "1"
+      - "2"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 50
+        }
+      }
+  "1":
+    id: "1"
+    taskid: 4478bd7c-8d70-42af-8f51-ab856b2afa0b
+    type: playbook
+    task:
+      id: 4478bd7c-8d70-42af-8f51-ab856b2afa0b
+      version: -1
+      name: WildFire - Detonate file
```

Comment:
Need to update that playbook to new conventions. No ${File.Entry} is past to the input